_posts/2020-03-30-exploting-spatial-diversity-using-unetstack3.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+comments: true
+title: Exploiting Dsitributed Spatial Diversity Using UnetStack
+date: 30/03/2020
+author: Prasad Anjangi
+categories: howto
+feature-img: "assets/img/sd-unet.jpg"
+thumbnail: "assets/img/sd-unet.jpg"
+tags: [howto, spatial diversity, cooperative diversity, robustness, data rate, performance boost]
+---
+
+Though it has been a buzzword over the past several years in the terrestrial radio frequency (RF) based wireless networks, much is not talked about practical spatial diversity systems in underwater wireless networks. It has proved and delivered tangible benefits to the end-user in the terrestrial wireless networks. Can we exploit this technique to make underwater wireless networks faster and more reliable and make that long-range communication link "just work" ? With the capability to exploit distributed spatial diversity, yes you can!
+
+### What's the key idea involved in distributed spatial diversity ?
+
+*Multiple communication nodes deployed at spatially distinct locations can receive independent copies of the same information. This group of receiver nodes can act as a combined spatial diversity receiver when they cooperate by sharing copies of information. We term this distributed spatial diversity. The framework supporting this patent-pending technique is implemented in UnetStack 3.1 and is referred to as `Unity`.*
+
+![Overview](../assets/img/sd.png)
```

Comment:
"Assisting Receiver" of the text in the diagram goes outside the oval. Can we fix that?

Reply:
Fixed.